woocommerce/woocommerce
# Add ratings, reviews and icons into Marketplace's Product Cards

### All Submissions:






### Changes proposed in this Pull Request:



The search on _WooCommerce > Marketplace_ page that shows product from the WooCommerce.com marketplace as cards. During previous re-design phase we added updated product cards in https://github.com/woocommerce/woocommerce/pull/30410.

This PR updates those product cards by:
- replacing a product image with icon (if it exists) that fits better
- mentioning a vendor who developed a product together with a link to reach the rest of their products on WooCommerce.com
- adding an average rating represented as 5 stars (golden, gray, or 50/50)
- adding the number of reviews left by customers for the product

Also, the text color for recurring payment cadence ("per year") was changed according to [the design](seb2o1DExPxoh5d53ftJif-fi-812%3A4611).

Closes 10867-gh-Automattic/woocommerce.com

### How to test the changes in this Pull Request:

1. Check out this branch to your local environment
2. Go to _WooCommerce > Marketplace_
3. Search for "memberships"
4. Ensure you see products like _WooCommerce Memberships_ that contain:
    - icon (instead of a product image), 
    - developed by ... (with a link to a vendor page on WooCommerce.com)
    - average rating represented as stars
    - number of reviews left by customers

<img width="1393" alt="Screen Shot 2021-09-30 at 11 07 48" src="https://user-images.githubusercontent.com/15897544/135419832-fc04a96f-7281-4f94-85e7-4da6731d57ff.png">

### Other information:




### Changelog entry

> Enhancement - Add ratings, reviews and icons into Marketplace's Product Cards.

### FOR PR REVIEWER ONLY:
